app/controllers/vendors_controller.rb

```diff
@@ -61,6 +61,30 @@ def update_vendor
   render json: response, status: :ok
 end
 
+
+def update_rating
+  body = JSON(request.body.read)
+  rating = body["rating"]
+  vendor_id = body["vendor_id"]
+  num = body["num"]
+  vendor = Vendor.find_by(id: vendor_id)
+  if vendor 
+    if vendor.rating
+      r = (vendor.rating).to_f * (vendor.num).to_f * 1.0 + (rating).to_f * 1.0
```

Comment:
nice, this looks like a great way to cumulate ratings

Reply:
TY